setup.py

```diff
@@ -14,12 +29,7 @@
         'pytest_testrail',
     ],
     package_dir={'pytest_testrail': 'pytest_testrail'},
-    install_requires=[
-        'pytest>=2',
-        'configparser>=3,<4',
-        'requests==2.11.1',
-        'simplejson'
-    ],
+    install_requires=requirements,
```

Comment:
Why is this better?  or is a formatting thing ?

Reply:
Didn't realize `configparser` in python3 was originally `ConfigParser` in python2, thought it was a third-party module. Updated `pytest_testrail.conftest` to do conditional import so this isn't needed anymore.